<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana_Medico</class>
 <widget class="QWidget" name="Ventana_Medico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>552</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Medico</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Ventana_paciente{
background-color: #fff
}

QPushButton#botonAceptar{
background-color: #0DA422;
border: 0;
}

QPushButton#botonCancelar{
background-color: #A40D0F;
color: #fff;
border: 0;
}</string>
  </property>
  <widget class="QLineEdit" name="campoDNI">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoApellido">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoNombre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="User">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAceptar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCancelar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFormMedico">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Nuevo Medico</string>
   </property>
  </widget>
  <widget class="QLabel" name="DNItex">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrese DNI:</string>
   </property>
  </widget>
  <widget class="QLabel" name="apellidotext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nombretext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Usertext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Passwordtext">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Tel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoTelefono">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Passwordtext_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Repetir Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>campoDNI</tabstop>
  <tabstop>campoApellido</tabstop>
  <tabstop>campoNombre</tabstop>
  <tabstop>campoTelefono</tabstop>
  <tabstop>User</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>Password2</tabstop>
  <tabstop>botonAceptar</tabstop>
  <tabstop>botonCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
